packages/frontend/src/features/sqlRunner/store/barChartSlice.ts

```diff
@@ -178,6 +164,38 @@ export const barChartConfigSlice = createSlice({
                 config.display.yAxis[index].position = position;
             }
         },
+        addYAxisField: (state) => {
+            if (!state.config) return;
+            if (!state.config.fieldConfig) return;
+
+            const yAxisFieldsAvailable = state.options.yLayoutOptions.filter(
+                (option) =>
+                    option.reference !==
+                    state.config?.fieldConfig?.y[0]?.reference,
```

Comment:
Shouldn't this be checking all the y fields not only `y[0]`

Reply:
Not too sure 🤷 I was thinking more like: choose whatever is not first - but I can update to check for all that are active!

Reply:
done and tested here: https://github.com/lightdash/lightdash/pull/10932/commits/2c0cb4807e8d50661bd103d26c0af93eab87c951